### PR TITLE
New workflow and readme update

### PR DESCRIPTION
**Pull request by Safi-Ullah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,25 +53,27 @@ Every newly submitted transaction should go on **the top of the list** and shoul
...
 2. Update the [`build`](package.json#L5) and [`start`](package.json#L6) scripts in [package.json](package.json) to respectively build and start your app.
 
 </details>
 
-### Get familiar with the API
-
-<details>
-<summary>Request examples</summary>
-
 ##### Get historical transactions
```
Comment (Safi-Ullah) [inline README.md:77]: @fenske,
What should be done to the API endpoints related to account transactions mentioned below?

Comment (fenske) [reply README.md:77]: No, this part we should leave as is as frontenders don't get the OpenAPI spec.

File: README.md [hunk 2]
```diff
@@ -53,25 +53,27 @@ Every newly submitted transaction should go on **the top of the list** and shoul
...
 
-### Import boilerplate
+### Import a boilerplate project
 
-Use [this link](https://docs.devskills.co/collections/85-the-interview-process/articles/342-importing-challenge-boilerplate) to get boilerplate code for your tech stack to configure a minimal setup for running the E2E tests.
+We have created a set of boilerplate projects for different tech stacks to help you get started quicker.
+
+To import a boilerplate project:
+
+1. Check out [this list](https://help.alvalabs.io/en/articles/7972852-supported-coding-test-boilerplates) to pick a desired boilerplate and copy its name (e.g., `backend-boilerplate-php-laravel`).
```
Comment (fenske) [inline README.md:64]: Could you please use `frontend-boilerplate-react-nextjs` as an example here?

**Review by fenske — CHANGES_REQUESTED** [review]
Added a couple of minor comments.

**Review by fenske — APPROVED** [review]
LGTM

**Pull request merged**